### DISCUSSION TITLE
Whe will it be available

### DISCUSSION THREAD

**Discussion by aXcELLcore** [AdaptiveMotorControlLab/CEBRA#4] [General]
Hello, I would like to know in what phase the project is, since I find it very interesting

**Comment by MMathisLab** [comment]
Hi there! Code will be available once paper is accepted, hopefully soon! In the meantime, we have a small group of beta testers, so reach out to mackenzie.mathis@epfl.ch if you want to test it.

**Reply by MMathisLab** [reply]
Update: the code will be released by April! Thank you all for your interest! ✨🦓

**Reply by wAOndering** [reply]
awesome thanks for the update!

**Comment by MMathisLab** [comment]
install docs: https://cebra.ai/docs/installation.html
pip: `pip install cebra `

paper comes out early May 🎉
